spras/mincostflow.py

```diff
@@ -127,4 +143,5 @@ def parse_output(raw_pathway_file, standardized_pathway_file):
 
         df = pd.read_csv(raw_pathway_file, sep='\t', header=None)
         df.insert(2, 'Rank', 1)  # adds in a rank column of 1s because the edges are not ranked
+        df = readd_direction_col_directed(df)
```

Comment:
The original paper using min cost flow on biological networks said the direction of flow was not meaningful. We should consider what directionality is appropriate for the output graph.

Reply:
The information is in the supplement https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2733244/bin/NIHMS109939-supplement-supp.pdf and the section is "Directionality of ResponseNet output". We could make a reference to that in the comments to explain why we treat the output as undirected. It states
> The flow algorithm we employ provides a directed network. However the directionality
of the interactions in the network is determined by the fact that we have connected all
genetic hits to the source of flow and all differentially expressed genes to the flow sink.
Therefore, except for the interactions between transcription factors and their targets, the
flow does not necessarily reflect a causal order of events